src/test/java/com/wikia/webdriver/elements/oasis/components/globalshortcuts/ActionExplorerModal.java

```diff
@@ -0,0 +1,115 @@
+package com.wikia.webdriver.elements.oasis.components.globalshortcuts;
+
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.core.elemnt.Wait;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ActionExplorerModal {
+
+  @FindBy(css = ".label-in-suggestions")
+  private WebElement searchSuggestions;
+
+  @FindBy(css = "#global_shortcuts_search_field")
+  private WebElement searchField;
+
+  @FindBy(css = ".global-shortcuts-search div[data-index='16']")
+  private WebElement specialAllPagesLink;
+
+  @FindBy(css = ".autocomplete-suggestions")
+  private WebElement autocompleteSuggestions;
+
+  private By actionExplorerModalSelector = By.cssSelector(".global-shortcuts-search");
+  private By keyboardShortcutsModalSelector = By.cssSelector(".global-shortcuts-help");
+
+  private Wait wait;
+  private Actions actions;
+  private JavascriptActions jsActions;
+  private WebDriver driver;
+
+  public ActionExplorerModal(WebDriver driver) {
+    this.wait = new Wait(driver);
+    this.actions = new Actions(driver);
+    this.jsActions = new JavascriptActions(driver);
+    this.driver = driver;
+
+    PageFactory.initElements(driver, this);
+  }
+
+  public ActionExplorerModal useShortcut(String shortcut) {
+    switch (shortcut) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement by "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1301)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)